bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Utils/Generators/ElectionGenerator.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ElectionGuard.Encryption.Utils.Generators
+{
+    public class TestElectionData
+    {
+        public Manifest Manifest { get; set; }
+        public InternalManifest InternalManifest { get; set; }
+        public CiphertextElectionContext Context { get; set; }
+        public ElGamalKeyPair KeyPair { get; set; }
+
+        public EncryptionDevice Device { get; set; }
+    }
+    public class ElectionGenerator
+    {
+        public static ElGamalKeyPair GenerateFakeKeyPair(bool useRandom = false)
+        {
+            var secretKey = useRandom ? BigMath.RandQ() : Constants.TWO_MOD_Q;
+            return ElGamalKeyPair.FromSecret(secretKey);
+        }
+        public static ElGamalKeyPair GenerateFakeKeyPair(ElementModQ secretKey)
+        {
+            return ElGamalKeyPair.FromSecret(secretKey);
+        }
+
+        public static CiphertextElectionContext GetFakeContext(
+            ElGamalKeyPair keyPair,
+            ElementModQ manifestHash)
+        {
+            return new CiphertextElectionContext(
+                1UL, 1UL,
+                keyPair.PublicKey,
+                Constants.TWO_MOD_Q,
+                manifestHash);
+        }
+
+        public static CiphertextElectionContext GetFakeContext(
+            ulong guardians, ulong quorum,
+            ElGamalKeyPair keyPair,
+            ElementModQ manifestHash)
+        {
+            return new CiphertextElectionContext(
+                guardians, quorum,
+                keyPair.PublicKey,
+                Constants.TWO_MOD_Q,
+                manifestHash);
+        }
+
+        public static CiphertextElectionContext GetFakeContext(
+            ulong guardians, ulong quorum,
+            ElementModP publicKey,
+            ElementModQ manifestHash)
+        {
+            return new CiphertextElectionContext(
+                guardians, quorum,
+                publicKey,
+                Constants.TWO_MOD_Q,
+                manifestHash);
+        }
+
+        public static EncryptionDevice GetFakeEncryptionDevice()
+        {
+            return new EncryptionDevice(12345UL, 23456UL, 34567UL, "Location");
+        }
+
+        /// <summary>
+        /// Generates fake election data
+        /// </summary>
+        public static TestElectionData GenerateFakeElectionData(
```

Comment:
i didnt clean it up everywhere, but as you make changes to files, go ahead and update the arrange statements in the unit tests to use this instead if you want.